contrib/systemd/containerized/origin-master.service

```diff
@@ -2,6 +2,8 @@
 After=docker.service
 Before=origin-node.service
 Requires=docker.service
+After=etcd.service
+Wants=etcd.service
```

Comment:
It wasn't clear to me either, since it didn't know where we were referencing these files (if anywhere) and what the documentation around them looked like.